tests/completion/lib.exp0

```diff
@@ -66,5 +66,5 @@ proc brexit {} {
 # tests to fail.
 spawn env -u PS1 bash --norc -i
 next
-send ". /usr/share/bash-completion/bash_completion; . ../../../data/completion/snap\n"
+send ". /usr/share/bash-completion/bash_completion; . ../../../data/completion/bash/snap\n"
```

Comment:
hmm so how did this even work before? perhaps we should have another:
```
send "set -e\n"
next
```
to ensure that errors result in failure?

Reply:
I think the reason that it was working is that the file was sourced again in the .exp files that use `lib.exp0`.

I am afraid that `set -e` will have unexpected effects in an interactive shell, because it is not a common thing to do. It is meant for non-interactive scripts. I do not want to debug that. Feel free to open a separate pull request.